msteams-platform/resources/teams-updates.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Microsoft Teams updates
+description: Learn about the latest updates to Microsoft Teams.
+author: v-ypalikila
+ms.author: lajanuar
+ms.localizationpriority: medium
+ms.topic: reference
+---
+# Microsoft Teams update
+
+[The new Microsoft Teams client](https://www.microsoft.com/en-us/microsoft-365/blog/2023/03/27/welcome-to-the-new-era-of-microsoft-teams/) is reimagined from the ground up with performance in mind. It's faster, simpler, smarter, and flexible to provide better experience for your apps and users. The new Teams client supports all the existing Teams app capabilities except Adaptive Card tabs. If you have an app that runs inside the Classic Teams, the app will most likely run in the new Teams client without any issues.
+
+The following are the advantages the new Teams client:  
+
+* The new Teams client uses the Evergreen version of Microsoft Edge WebView2 to ensure Teams client is always up to date with the latest fixes and improvements available in Microsoft Edge and Chromium.
+
+* The new Teams client has been rebuilt from the ground up with performance in mind and includes all the platform infrastructure responsible for bootstrapping your app and powering the SDK APIs that it uses.  
+
+You can use the following property to identify your app usage in the new Teams or Classic Teams client:
+
+* For TeamsJS v1.x: [`hostName`](/javascript/api/@microsoft/teams-js/hostname?view=msteams-client-js-latest&preserve-view=true)
+* For TeamsJS v2.x: [`AppHostInfo`](/javascript/api/@microsoft/teams-js/app.appinfo?view=msteams-client-js-latest&preserve-view=true#@microsoft-teams-js-app-appinfo-host)
+
+The new Teams or Classic Teams client are represented using the `teams` and `teamsModern` fields, respectively.
+
+## Timelines and rollout
+
+To ensure a smooth transition, a phased rollout of the new platform is planned as follows:
+
+* **Developer Preview**: The new Teams client is available in Public Developer Preview starting June 2023. You can access the new platform and test your apps. We encourage you to adopt the feature early and provide feedback to help refine the platform.
+
+* **Availability of all platform features from Classic Teams**: All the platform features from Teams classic will be available in the new Teams client by August 2023.
+
+## Known issues
```

Comment:
Sideloading of apps is not supported yet in the new Teams client. Developers can sideload their app in Classic Teams client and use it in new Teams client. This support will be available by end of September.

Reply:
Added the limitation:

_Sideloading of apps isn't supported in the new Teams client. You can sideload an app in the Classic Teams client and use it in new Teams client._

As per guidelines, we don't mention future dates or call out any support in the future. We would update the docs once the limitation is available in the new Teams client.